xml/System/DateTime.xml

```diff
@@ -46,351 +46,351 @@
+[!code-vb[System.DateTime.Persistence#2](~/samples/snippets/visualbasic/System.DateTime/Persistence.vb#2)]  
+
+#### Persisting values as integers  
+
+You can persist it as an <xref:System.Int64> value that represents a number of ticks. In this case, you don't have to consider the culture of the systems the <xref:System.DateTime> values are persisted and restored on.  
   
- To persist a <xref:System.DateTime> value as an integer:  
+To persist a <xref:System.DateTime> value as an integer:  
   
 -   If the <xref:System.DateTime> values represent single moments in time, convert them to UTC by calling the <xref:System.DateTime.ToUniversalTime%2A> method.  
-  
 -   Retrieve the number of ticks represented by the <xref:System.DateTime> value from its <xref:System.DateTime.Ticks%2A> property.  
   
- To restore a <xref:System.DateTime> value that has been persisted as an integer:  
+To restore a <xref:System.DateTime> value that has been persisted as an integer:  
   
 1.  Instantiate a new <xref:System.DateTime> object by passing the <xref:System.Int64> value to the <xref:System.DateTime.%23ctor%28System.Int64%29> constructor.  
+1.  If the <xref:System.DateTime> value represents a single moment in time, convert it from UTC to the local time by calling the <xref:System.DateTime.ToLocalTime%2A> method.  
+  
+The following example persists an array of <xref:System.DateTime> values as integers on a system in the U.S. Pacific Time zone. It restores it on a system in the UTC zone. The file that contains the integers includes an <xref:System.Int32> value that indicates the total number of <xref:System.Int64> values that immediately follow it.  
   
-2.  If the <xref:System.DateTime> value represents a single moment in time, convert it from UTC to the local time by calling the <xref:System.DateTime.ToLocalTime%2A> method.  
+[!code-csharp[System.DateTime.Persistence#3](~/samples/snippets/csharp/System.DateTime/Persistence.cs#3)]
+[!code-vb[System.DateTime.Persistence#3](~/samples/snippets/visualbasic/System.DateTime/Persistence.vb#3)]  
+
+#### Serializing DateTime values  
+
+You can persist <xref:System.DateTime> values through serialization to a stream or file, and then restore them through deserialization. <xref:System.DateTime> data is serialized in some specified object format. The objects are restored when they are deserialized. A formatter or serializer, such as <xref:System.Xml.Serialization.XmlSerializer> or <xref:System.Runtime.Serialization.Formatters.Binary.BinaryFormatter>, handles the process of serialization and deserialization. For more information about serialization and the types of serialization supported by the .NET Framework, see [Serialization](~/standard/serialization/index.md).
+  
+The following example uses the <xref:System.Xml.Serialization.XmlSerializer> class to serialize and deserialize <xref:System.DateTime> values. The values rep
```

Comment:
The Remarks section (the head) is not rendering. the mini TOC that follows is. But effectively, all of this content is not discoverable.
I'm not sure why, but one possible explanation is that in the markdown, the parser expects Remarks to precede Examples, since that's the way the DDUEML was formatted, and possibly how EXMAXML expects mref to be formatted. 
//cc @dend